scripts/dts/edtlib.py

```diff
@@ -1278,6 +1311,37 @@ def __repr__(self):
         return "<ControllerAndData, {}>".format(", ".join(fields))
 
 
+class PinCtrl:
+    """
+    Represents a single pin configuration on a device, e.g. pinctrl-0 or
```

Comment:
"single pin configuration" is misleading. It is a single pin control configuration that may well control several pins (not only a single pin).

Represents a pin control configuration for a (set of) pins on a device, e.g. pinctrl-...

Reply:
Updated.